client/analytics/settings/index.js

```diff
@@ -116,6 +116,9 @@ class Settings extends Component {
 	 * @param {object} state - State
 	 */
 	persistChanges( state ) {
+		// TODO: Should remove global state from the file. This creates
```

Comment:
We usually use `@todo` for to-do comments (see #1474):
```suggestion
		// @todo Should remove global state from the file. This creates
```